src/Kafka/CreateKafkaInstanceWithSizes/Stories/storiesHelpers.tsx

```diff
@@ -50,76 +49,113 @@ export const AZURE: ProviderInfo = {
 
 export const PROVIDERS: Providers = [AWS, AZURE];
 
-const SIZES: { [key: string]: Size[] } = {
-  aws: [
-    {
-      id: "x1",
+const SIZES: { [key: string]: GetSizesData } = {
+  aws: {
+    standard: [
+      {
+        id: "x1",
+        displayName: "1",
+        quota: 1,
+        status: "stable",
+        ingress: 3,
+        egress: 31,
+        storage: 5,
+        connections: 6,
+        connectionRate: 7,
+        maxPartitions: 8,
+        messageSize: 9,
+        trialDurationHours: undefined,
+      },
+      {
+        id: "x2",
+        displayName: "2",
+        quota: 2,
+        status: "preview",
```

Comment:
Similar to my other comment, I'm not sure how this data relates to how we will use this component in production, but looking at the [current implementation](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/managed-services/cicd/saas/saas-kas-fleet-manager.yaml#L311), this value is provided as `maturityStatus`.